helix-core/src/main/java/org/apache/helix/BaseDataAccessor.java

```diff
@@ -257,4 +257,9 @@
    * reset the cache if any, when session expiry happens
    */
   void reset();
+
+  /**
+   * Close the opened zkConnection
```

Comment:
Please remove the mention of ZK-related things in this interface. BaseDataAccessor is a generic interface - it shouldn't assume the use of ZK.

Something like "Close the connection to the metadata store" would be more appropriate.

Reply:
done.